plugins/gdk-pixbuf/pixbufloader-jxl.c

```diff
@@ -349,7 +340,7 @@ static void draw_pixels(void *context, size_t x, size_t y, size_t num_pixels,
                 decoder_state->pixel_format.num_channels * x +
                 gdk_pixbuf_get_rowstride(output) * y;
 
-  cmsDoTransform(decoder_state->transform, pixels, dst, num_pixels);
+  memcpy(dst, pixels, num_pixels * decoder_state->pixel_format.num_channels);
```

Comment:
It should be easier (and faster) to use the `JxlDecoderSetImageOutBuffer()` instead of using the callback with `JxlDecoderSetImageOutCallback()`.

Reply:
Isn't this issue a bit off-topic though? i.e. should that behavior be changed in a separate, unrelated commit?

Reply:
If you have a strong opinion, this could be done in a separate commit, but since the simplification is possible because of the change of not doing the sRGB conversion, I would also be happy having it part of this one.

Reply:
SGTM

Reply:
Let's submit this now and then change to not using the callback in a follow up pull request